### DISCUSSION TITLE
organizing code based on language.

### DISCUSSION THREAD

**Discussion by AkashKishore-Mukhia** [Sagar0-0/DSA#370] [Ideas]
hey, sagar i would like to contribute to the project, since this project has solution on different languages i like to organize them by languages, so if you like the idea please reply to this issue and also share you though if you have any.

**Comment by Sagar0-0** [comment]
Hi Akash, thanks for sharing your idea. As the project is very huge now, it will be nearly impossible to move all files to their language-based directories. And also for the sake of simplicity, I also want beginners to just go and directly contribute to the repository without handling all the directories kinds of stuff. I hope you understand my point, feel free to share more ideas to maintain this repository in a more structured way! Thank you.

**Comment by Sagar0-0** [comment]
![2022-09-26 10 30 20](https://user-images.githubusercontent.com/85388413/192197242-969d172e-0e96-43bd-959c-cba7d1b35f52.png)
Was not supposing that!!

**Comment by AMANSAINI7144** [comment]
hey, Sagar I would like to contribute today's potd code.
can I submit today's code?
